test/unit/storage_sync_test.rb

```diff
@@ -38,23 +40,16 @@ def test_redis_malformed_url
     end
   end
 
-  def test_redis_url_without_scheme
-    assert_nothing_raised do
-      StorageSync.send :new, url('foo')
-    end
-  end
```

Comment:
we don't support urls without a schema anymore, correct?

Reply:
Apparently we do. I don't know why I removed that test. Fixed in https://github.com/3scale/apisonator/pull/384/commits/461a712c4ed8242de9d61e66119b5dd17c6b5a3d